slackbridge/bots.py

```diff
@@ -91,10 +92,12 @@ def check_slack_rtm(self):
 
 class UserBot(IRCBot):
 
-    def __init__(self, nickname, realname, user_id, channels):
+    def __init__(self, nickname, realname, user_id, users, channels):
+        self.topics = {}
         self.nickname = '{}-slack'.format(utils.strip_nick(nickname))
         self.realname = realname
         self.user_id = user_id
+        self.users = users
```

Comment:
I think instead of keeping this as a local variable in each bot that's it's probably better to put this as a class variable under `IRCBot` so that the bridge bot and all the user bots have access to the same data.